### PR TITLE
CN 3772 community screen improvements

### PR DESCRIPTION
**Pull request by srkela**
PR Checklist:
- [x] I have named the PR correctly
- [x] I have added at least two reviewers
- [x] I have assigned the PR to myself
- [x] I have merged develop into my feature branch and resolved conflicts
- [x] I have tested all new features locally
- [x] I have tested relevant flow locally

### How to name a PR
- If it includes only one issue [CN-1234 Issue name]
- If it includes many issues [CN-1234, CN-2345, CN-3456 What was updated/changed/added]
- If the PR is not ready yet, add [DRAFT] in front of the name

### How to test locally
1. Test happy path
2. Test other paths
3. Remove all warnings

### How to relevant flow locally
- If working on part of CelPay, check the whole happy path

### CODE REVIEW

File: app/redux/community/communityReducer.js [hunk 1]
```diff
@@ -9,14 +9,12 @@ const initialState = {
 
 export default function communityReducer(state = initialState, action) {
 
-    switch (action.type) {
-      case ACTIONS.GET_COMMUNITY_STATISTICS_SUCCESS:
+    if (action.type === ACTIONS.GET_COMMUNITY_STATISTICS_SUCCESS) {
```
Comment (fjovakaric) [inline app/redux/community/communityReducer.js]: Do we need this?

File: app/components/screens/Community/Community.js [hunk 2]
```diff
@@ -37,78 +33,288 @@ class Community extends Component {
...
+          </>
+      )
+  }
+
   render() {
-    const { communityStats, celStats, ethStats, btcStats } = this.props;
+    const { communityStats } = this.props;
     const style = CommunityStyle();
 
-    if (!communityStats) {
```
Comment (fjovakaric) [inline app/components/screens/Community/Community.js]: With this you dont need `communityStats &&` anywhere
Should this return a `<LoadingScreen />` component

File: app/redux/community/communityReducer.js (hunk 1 above)
Comment (srkela) [reply app/redux/community/communityReducer.js]: reverted

File: app/components/screens/Community/Community.js (hunk 2 above)
Comment (srkela) [reply app/components/screens/Community/Community.js]: fixed

**Pull request merged**